README.md

```diff
@@ -0,0 +1,128 @@
+# Mountsion CBS Secondary Website
+
+This project is a website that is being made by 4 students from Mountsion CBS Secondary along with the help of RedHat Waterford
```

Comment:
I think we should reword this as yes it is currently 4 students, but this will be an ongoing project involving students from your school, Also we can acknowledge those who contribute in a section further down the ReadMe